assets/js/screens/Project/Project.jsx

```diff
@@ -8,14 +8,52 @@ import Error from "../../components/UI/Error";
 import Loading from "../../components/UI/Loading";
 import { useQuery } from "@apollo/react-hooks";
 import AddOutlineIcon from "../../../css/fonts/zondicons/add-outline.svg";
-import { GET_PROJECT } from "../../components/Project/project.query";
+import {
+  GET_PROJECT,
+  INGEST_SHEET_STATUS_UPDATES_FOR_PROJECT_SUBSCRIPTION
+} from "../../components/Project/project.query";
+import gql from "graphql-tag";
 
 const ScreensProject = ({ match }) => {
   const { id } = match.params;
-  const { loading, error, data } = useQuery(GET_PROJECT, {
+  const { loading, error, data, subscribeToMore } = useQuery(GET_PROJECT, {
     variables: { projectId: id }
   });
 
+  const handleIngestSheetStatusChange = (prev, { subscriptionData }) => {
+    if (!subscriptionData.data) return prev;
+
+    const ingestSheet = subscriptionData.data.ingestSheetUpdatesForProject;
+    console.log(ingestSheet);
```

Comment:
Unless we want to keep these console.logs as helpers, I'd wipe 'em out before merging.